lib/include/chunk.h

```diff
@@ -12,7 +12,8 @@
 #define BLOCK_BUFFER_SIZE (CHUNK_SIZE*CHUNK_SIZE*CHUNK_SIZE*BLOCK_SIZE)
 #define BLOCKS_HEADER_SIZE 2
 
-namespace konstructs {
+namespace konstructs 
+{
```

Comment:
Yeah np, will put them back and send a new pull request. IDE gets trigger happy.